tests/e2e/tools/lib/editor.js

```diff
@@ -87,7 +85,7 @@ const searchAndInsertBlock = async ({page}, blockName, namespace = '') => {
     return await page.locator(`button.editor-block-list-item-${namespace.toLowerCase()}[role="option"]`).click();
   }
 
-  return await page.getByRole('option', {name: blockName}).click();
+  return await page.getByRole('option', {name: blockName, exact: true}).click();
```

Comment:
Have you tested this new attribute wi the other specs ?

Reply:
No, I admit I haven't. But for the `Video` block it's mandatory because there is also a `VideoPress` block... or maybe I could try using the namespace instead 🤔

Reply:
Is there any siginifican reason to add it? 

I've tested with this new attribute and it seems to break all [these specs](https://github.com/greenpeace/planet4-master-theme/pull/2419).

Reply:
<img width="1261" alt="Screen Shot 2024-11-01 at 09 33 25" src="https://github.com/user-attachments/assets/770a55bb-92bf-49de-aa9b-8c0670d6ad2b">

Reply:
Without `exact = true`

<img width="892" alt="Screen Shot 2024-11-01 at 09 36 54" src="https://github.com/user-attachments/assets/9567fabb-8307-46de-94cc-9583683945de">

Reply:
I've explained to you why I added it [here](https://github.com/greenpeace/planet4-master-theme/pull/2420#discussion_r1825685259)
As I also said, I could try using the namespace instead